### PR TITLE
Use static boost libraries

### PR DESCRIPTION
**Pull request by as6520**
Use static boost libraries for static kwiver build

### CODE REVIEW

File: CMake/kwiver-depends-Boost.cmake [hunk 1]
```diff
@@ -3,6 +3,9 @@ if (NOT DEFINED KWIVER_BOOST_VERSION)
   set(KWIVER_BOOST_VERSION 1.54)
 endif()
 
+if (NOT ${KWIVER_BUILD_SHARED})
```
Comment (dstoup) [inline CMake/kwiver-depends-Boost.cmake]: I think the decision about this block has less to do with whether kwiver is built static and more to do with whether Boost was.  It think we will need something in the fletchConfig.cmake (if it's not already there) that says which way Boost was built.

Comment (as6520) [reply CMake/kwiver-depends-Boost.cmake]: Moved the change to fletch in https://github.com/Kitware/fletch/pull/556

**Comment by mwoehlke-kitware** [comment]
Should this PR be closed?

**Comment by as6520** [comment]
Yeah addressed in kitware/fletch#556

**Pull request closed without merging**